src/main/app/claims/offerModelConvertor.ts

```diff
@@ -0,0 +1,16 @@
+import { Offer as OfferForm } from 'offer/form/models/offer'
+import { Offer } from 'claims/models/offer'
+
+export class OfferModelConverter {
+
+  static convert (offerForm: OfferForm): Offer {
+    if (offerForm && offerForm.offerText) {
```

Comment:
is there a scenario where this check is needed?

Reply:
yes it does, what do you think?

Reply:
when?
It seems strange that you would be returning undefined when you are about to save an offer?

Reply:
Yes, reaching this point means that the `Offer` has already passed validation and we're sure that `offerText` is filled in. If for some reason `offerForm` would be undefined here then the app would crash here and indicate a bug somewhere else.

Returning "blank" values instead of failing fast can lead to hard to find bugs.